Controller/CategoryAdminController.php

```diff
@@ -99,7 +98,7 @@ public function treeAction(Request $request)
         $datagrid = $this->admin->getDatagrid();
 
         if ($this->admin->getPersistentParameter('context')) {
-            $datagrid->setValue('context', ChoiceType::TYPE_EQUAL, $this->admin->getPersistentParameter('context'));
+            $datagrid->setValue('context', null, $this->admin->getPersistentParameter('context'));
```

Comment:
Yes, I passed there `EqualType::TYPE_IS_EQUAL` and `EqualType::TYPE_IS_NOT_EQUAL` - no errors, everything works. Also if i pass there `ChoiceType::TYPE_CONTAINS` - there will be no errors too, because it it the same as to pass there `1`.  But is is incorrect to use consts from other types, even if they are equal.